### PR TITLE
Fix ValueRange.from_json() method

### LINKED ISSUE #780: exception for worksheet.get('range') on blank cells
**Important**: Please do not post usage questions here. 
To get a quick response, please ask a question on Stack Overflow using `gspread` tag.
See existing questions: https://stackoverflow.com/questions/tagged/gspread

---

**Describe the bug**
using the get method with a valid range on a worksheet but the range is empty cells.

**To Reproduce**
1)  Authorise gspread and open spread sheet in browser
2) Read cells using get method with a range --> .get('A3:A19'.
 At this point all is fine. Values in cells are returned in a list
3) Delete the values in the spreadsheet range
4) Run code again.
5) 

**Expected behavior**
Returns an empty list or a None

**Code example***
If applicable, provide a code example to help explain your problem.
import gspread
from google.oauth2.service_account import Credentials
scope = [
    'https://www.googleapis.com/auth/spreadsheets',
    'https://www.googleapis.com/auth/drive.file'
              ]

credentials = Credentials.from_service_account_file(
    filename='G:\\PycharmProjects\\DebtPayDown\\JSON\\PayDebtsDown.json',
    scopes=scope)

gc = gspread.authorize(credentials)

sh          = gc.open_by_key('<my_sheet_id>')
work_sheet  = sh.worksheet("PayThese")
result      = work_sheet.get('A3:A19')

where A3:A19 are blank cells.
works fine if the cells are not blank / contain values (strings in this case).

**Screenshots**

**Environment info:**
Win8.1
Python 3.8
gspread 3.6

**Stack trace or other output that would be helpful**
File "G:/PycharmProjects/DebtPayDown/arst.py", line 18, in <module>
    result      = work_sheet.get('A3:A19')
  File "E:\Python38\lib\site-packages\gspread\utils.py", line 592, in wrapper
    return f(*args, **kwargs)
  File "E:\Python38\lib\site-packages\gspread\models.py", line 965, in get
    return ValueRange.from_json(response)
  File "E:\Python38\lib\site-packages\gspread\models.py", line 51, in from_json
    new_obj = cls(json['values'])
KeyError: 'values'

**Additional context**
Love your work, thank you VERY much for creating this :)

### PR DESCRIPTION
**Pull request by erakli**
Closes  #780

### CODE REVIEW

**Comment by erakli** [comment]
@burnash, please, take a look on this PR

**Comment by burnash** [comment]
Thank you very much for the PR and sorry for the late reply.

File: gspread/models.py [hunk 1]
```diff
@@ -48,7 +48,8 @@
 class ValueRange(list):
     @classmethod
     def from_json(cls, json):
-        new_obj = cls(json['values'])
+        values = json['values'] if 'values' in json else []
```
Comment (burnash) [inline gspread/models.py:51]: I think we can improve it even further if we use `dict.get` with default, e.g. `json.get('values', [])`

Comment (erakli) [reply gspread/models.py]: Great idea, done in 903f268

**Comment by burnash** [comment]
Thank you very much!

**Pull request merged**